src/components/Board.js

```diff
@@ -0,0 +1,28 @@
+import "./Board.css";
+import Square from "./Square";
+
+function Board(props) {
+  return (
+    <div>
+      <h1>Tic Tac Toe</h1>
+      <div className="status">{props.turn}</div>
```

Comment:
We also can declare
`const { turn, squares, onClick, onRestart } = props`
before return and use `<div className="status">{turn}</div>` here